src/screens/gm/tweak.cpp

```diff
@@ -741,11 +833,65 @@ void GuiShipTweakPlayer2::onDraw(sf::RenderTarget& window)
     can_launch_probe->setValue(target->getCanLaunchProbe());
 }
 
+void GuiShipTweakCpu::onDraw(sf::RenderTarget& window)
+{
+    // Update repair rate.
+    repair_rate_slider->setValue(target->getAutoRepairRate());
+}
+
+void GuiStationTweakBase::onDraw(sf::RenderTarget& window)
+{
+    hull_slider->setValue(target->hull_strength);
+    // TODO
```

Comment:
That is never a good sign...

Reply:
I haven't cracked tweaking station shields yet